gradle.properties

```diff
@@ -1,7 +1,9 @@
 #environment
 environmentName=GUI
+browserName=chrome
 browserWidth=414
 browserHeight=736
+threadCount=3
```

Comment:
And I suggest you change this threadCount=`3` to `6`
because we have the problem of not connecting, when logging in with the same account in the same browser

Reply:
done